scripts/install/RegisterQtiPackageExporter.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA ;
+ */
+declare(strict_types=1);
+
+namespace oat\taoQtiTest\scripts\install;
+
+use common_report_Report;
+use oat\oatbox\filesystem\FileSystemService;
+use oat\tao\helpers\FileHelperService;
+use oat\taoQtiTest\helpers\QtiPackageExporter;
+use taoQtiTest_models_classes_export_TestExport22;
+use oat\oatbox\extension\InstallAction;
+
+class RegisterQtiPackageExporter extends InstallAction
+{
+    public function __invoke($params)
+    {
+        $serviceManager = $this->getServiceManager();
+        $qtiPackageExporter = new QtiPackageExporter(
+            new taoQtiTest_models_classes_export_TestExport22(),
```

Comment:
What is this class?

Reply:
It looks like this class extends test export class to add ability to export QTI 2.2 (original PR where it was created - https://github.com/oat-sa/extension-tao-testqti/pull/544) Previously it was instantiated directly in QtiPackageExporter here - https://github.com/oat-sa/extension-tao-testqti/pull/1827/files#diff-4995a42cda3b7d9303f573b95c6d790cL57. I added it as a constructor dependency in this script to be able to write tests.